module/app/upgrades/register.go

```diff
@@ -23,7 +21,6 @@ func RegisterUpgradeHandlers(
 	bankKeeper *bankkeeper.BaseKeeper, bech32IbcKeeper *bech32ibckeeper.Keeper, distrKeeper *distrkeeper.Keeper,
 	mintKeeper *mintkeeper.Keeper, stakingKeeper *stakingkeeper.Keeper, upgradeKeeper *upgradekeeper.Keeper,
 ) {
-	fmt.Println("Enter RegisterUpgradeHandlers")
```

Comment:
I removed a couple of these to prevent unnecessary logging on various gravity commands.